pkg/apis/config/v1alpha2/multi_cluster_service.go

```diff
@@ -0,0 +1,73 @@
+package v1alpha2
```

Comment:
Why is the multicluster crd changing ?

Reply:
Check https://github.com/openservicemesh/osm/pull/4421#discussion_r791951218